docs/dev/plugins/cloudFoundry/deploy-cf-bosh-lite.md

```diff
@@ -0,0 +1,161 @@
+# Deploying CloudFoundry using BOSH-Lite
+
```

Comment:
I think for the non cf pros it would be useful to add a short reason why we need BOSH-Lite